toolkits/global/packages/global-ratings-survey/scss/50-components/_ratings-survey.scss

```diff
@@ -10,7 +10,11 @@
 	justify-content: center;
 }
 
-.c-ratings-survey__heading {
+// Specificity score increased to ensure overrides heading styles on all of our imprint sites
+.c-ratings-survey h1.c-ratings-survey__heading,
```

Comment:
@benjclark I'm thinking `!important` is better than making the selector somewhat more complex?